recipe/meta.yaml

```diff
@@ -11,21 +11,20 @@ source:
   sha256: 69e9fa08bacb2475e636da05a6c222b17c67f1ebeab3793762062248dd9d842f
 
 build:
-  number: 201
+  number: 202
   features:
     - blas_{{ variant }}
   skip: true  # [win]
 
 requirements:
   build:
     - toolchain
-    - gcc
+    - gcc  # [unix]
     - cmake
     - blas 1.1 {{ variant }}
     - openblas 0.2.19|0.2.19.*
 
   run:
-    - libgfortran
```

Comment:
I do not agree : arpack does not link to libgfortran on it own, if openblas stopped linking to libgfortran and likning to something else the package would include a useless dependency.